react/components/Table/util/getRowHeight.js

```diff
@@ -0,0 +1,12 @@
+export default density => {
```

Comment:
Why not putting all these functions in a `utils.js` or `commons.js` or whatever instead of creating all these files?

Reply:
Just my personal preference. I do not like a huge js file with a lot of utils. But if the majority of people do not like it I could change. (doesn't really matter).

Reply:
Done